File: Python_Assignment_/Python_Programming_Assignment_2/Assignment2_5.py
# 5. Write a program which accept one number for user and check 
#    whether number is prime or not

# Input : 5                     Output : It is a prime number

# To check whether given number is prime or not
def Prime(No):
    if No < 2:
        print("Please enter valied input, 1, 0 and negative numbers not allowed")
        return
    
    else:
        flag = 0
        for i in range(2, int(No/2)+1):
            if No%i == 0:
                flag = 1
                break

        if flag == 0:
            print("It is Prime Number")

        else:
            print("It is not Prime Number")
        

def main():
    no = int(input("Enter number to check for Prime : "))
    Prime(no)

# Starter
if __name__ == "__main__":
    main()

File: Typescript_Assignment/Typescript_Assignment2/Assignment2_5.ts
// 2.5 Write a typescript program which contains one function named ChkString. 
//     That function accept one string and check whether that string contains
//     "Marvellous" word or not.

// Input : "Pune Kothrud Marvellous Infosystems"
// Output : String contains Marvellous in it.


function ChkString(str : string, str1 : string) : boolean
{
    // split the given sentence into words and store them into array
    var Data : string[] = str.split(" ")

    return (Data.indexOf(str1) !==-1)
}


var sentence: string = "Pune Kothrud Marvellous Infosystems"
var word: string = "Marvellous"

var Result : boolean = ChkString(sentence,word)

if(Result == true)
{
    console.log(`String contains "${word}" in it.`)
}

else
{
    console.log(`String does not contains "${word}" in it.`)
}

/*

E:\Typescript\Assignment\Typescript_Assignment2>tsc Assignment2_5.ts

E:\Typescript\Assignment\Typescript_Assignment2>node Assignment2_5.js
String contains "Marvellous" in it.

E:\Typescript\Assignment\Typescript_Assignment2>

*/